modules/ROOT/pages/api-led-develop.adoc

```diff
@@ -0,0 +1,121 @@
+
+The steps in this section assume you have launched Studio before each Step.
+
+=== Step 1: Add Your Anypoint Platform Credentials to Studio
+
+Add your Anypoint Platform credentials in Studio once, and they'll be available for all the projects you create.
+
+. In Studio, open the *Preferences* dialog.
+. Select *Anypoint Studio > Authentication*.
+. Enter your Anypoint Platform username and password.
+. Click *Sign in*.
+. Click *Apply and Close*.
+
+These credentials are available for the duration indicated when you entered them, seven days by default.
+
+=== Step 2: Create a New Project with an Imported API Specification
+
+To deploy an API to the Mule runtime engine in the target of your choice, 
+build the interface and implementation in a Mule app using Studio. In this step,
+we'll create the project.
+
+. In Studio, select *File > New > Mule Project*.
+. In the dialog box, provide the following values:
+  ** *Project Name*: Hello-World
+  ** *Runtime*: If there is a choice, select the latest version of Mule. This is the Mule runtime engine that is hosted within Studio for testing.
+  ** Select the *From Design Center or local files* tab.
+    . Click the button to the right of the *Location* field.
+    . Select *Design Center*.
+    . If the correct username for your Anypoint Platform org appears, you'll see a list of API specifications.
+    . Choose *Hello World*.
+    . Click *OK*.
+    . Make sure the checkbox *Scaffold flows from these API specifications* is selected.
+    . To verify the import, check Package Explorer for `hello-world/src/main/resources/api/hello-world.raml`.
+
+Studio opens the *hello-world.xml* XML file,
+which contains some of the basic elements you'll need for the API interface. Next, we'll configure these elements,
+and later on we'll create the business logic in a separate XML file for the implementation. 
+
+=== Step 3: Configure the RESTful API Interface
+
+Using the scaffolding provided by Studio, complete the RESTful API interface:
+
+. In Studio, right-click `hello-world/src/main/mule/hello-world.xml` and select *Refactor > Rename*.
+. Rename the file: `interface.xml`.
+. Configure the main listener:
+  .. In the canvas, click the *hello-world-main* Listener to display the General configuration tab.
+  .. Click the green plus sign to create a global configuration for HTTP listeners in this project.
+  .. Verify that the host is `0.0.0.0` and the port is `8081`.
+  .. Click *Test Connection*, and then click *OK* when the connection is successful.
+  .. If you click *Global Elements* at the bottom of the canvas, you'll see this configuration is a global configuration.
+. Click *Configuration XML* at the bottom of the canvas to see the XML that has been created.
+
+Once we have an implementation set up, we'll return to `interface.xml`.
+
+=== Step 4: Create the RESTful API Implementation
+
+Find and assemble the elements needed to define the business logic of your API:
+
```

Comment:
I think you mean why are we using both terms for the same thing? Because not everyone knows that implementation means "business logic" in this context (some use it to mean the whole API). I may have said it too often?